### PR TITLE
Feature lwdev 10071 stellar kin settings segregation

### PR DESCRIPTION
**Pull request by OttoVT**

### CODE REVIEW

File: src/Lykke.Service.Stellar.Api.Core/Domain/Asset.cs [hunk 1]
```diff
@@ -15,6 +15,6 @@ public class Asset
         public string TypeName { get; }
         public int Accuracy { get; }
 
-        public static Asset Stellar { get; } = new Asset("XLM", "", "Stellar Lumen", "native", 7);
+        //public static Asset Stellar { get; } = new Asset("XLM", "", "Stellar Lumen", "native", 7);
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Service.Stellar.Api.Core/Domain/Asset.cs]: Remove commented code

File: src/Lykke.Service.Stellar.Api.Services/Balance/BalanceService.cs [hunk 2]
```diff
@@ -59,12 +76,28 @@ public bool IsAddressValid(string address)
...
 
+
+            foreach (var @char in address)
+            {
+                if (_forbiddenAddressChars.Contains(@char))
+                {
+                    containsForbiddenChar = true;
+                    break;
+                }
+            }
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Service.Stellar.Api.Services/Balance/BalanceService.cs]: ```containsForbiddenChar = address.Any(_forbiddenAddressChars.Contains);``` ?

File: src/Lykke.Service.Stellar.Api.Core/Domain/Asset.cs (hunk 1 above)
Comment (OttoVT) [reply src/Lykke.Service.Stellar.Api.Core/Domain/Asset.cs]: fixed

File: src/Lykke.Service.Stellar.Api.Services/Balance/BalanceService.cs (hunk 2 above)
Comment (OttoVT) [reply src/Lykke.Service.Stellar.Api.Services/Balance/BalanceService.cs]: fixed. You are right!)

**Pull request merged**